versioneer.py

```diff
@@ -1643,7 +1643,7 @@ class VersioneerBadRootError(Exception):
     """The project root directory is unknown or missing key files."""
 
 
-def get_versions(verbose=False):
+def get_versions(verbose=False, root=None):
```

Comment:
I think this file was copied over from another repo,
maybe @IsaacYangSLA can help comment on this